### ISSUE TITLE
Folders not Rendering in FVTT 0.8.9... sometimes

### ISSUE THREAD

**Issue by Zephyr2011** [earlSt1/vtt-compendium-folders#107]
**Describe the bug**
Compendium Folders are not rendering within compendiums in FVTT 0.8.9 with version 2.3.53. This seems to be limited to worlds and instances that did not view a compendium with folders in FVTT 0.8.9 with a previous version of CF?

Folders still exist, if CF is disabled, the #CF_Entity entries become visible.

**To Reproduce**
1) Use fresh instance of FVTT 0.8.9 with libwrapper and CF active (also works w/o libwrapper)
2) Create new compendium
3) Create folder within compendium. Folder does not render. "blinks" briefly then is hidden.

3a) Create new folder in actors/items/scenes directory
4a) Create new actor/item/scene within new folder
5a) export folder structure to compendium from 2. Folder does not render, actor/item/scene appears

If "Auto Create folders on Import" is selected and an entity is within a (hidden) folder in a compendium, the folders will be imported. Purely a rendering issue.

This also appears to apply to compendiums from modules. Instance that used compendium folders before 0.8.9 did not exhibit the same behavior. Unsure how to replicate without risking important data.

**Expected behavior**
When a new folder is created within a compendium, the folder should remain visible.

When a folder is created in a directory and exported to a compendium, the folder should remain visible.

When a module includes compendiums that use CF, the folders should be visible.

**Screenshots**
I can post a video if helpful, basically the folders are just blank.

**Information**
- FoundryVTT Version: 0.8.9
- Compendium Folders Version: 2.3.53
- Game System + Version: DnD5e - Fifth Edition System v1.4.2
- All other modules disabled? yes

**Comment by Zephyr2011** [comment]
UPDATE:

tl;dr: I've resolved my issue by uninstalling and reinstalling Foundry using the latest install link rather than an update path.

I swapped data from another install of Foundry that was working as expected (ie: folders displayed properly). The installs that were having the rendering issues _still_ had the folders hidden, despite the data being directly copied from properly working installs. I renamed the data folders, uninstalled Foundry, and reinstalled using the 0.8.9 installer, rather than an upgrade path. All seems to be working properly now. 

No idea what got messed up on the installs previously, very strange that it affected 2 installs on 2 different machines, but I don't think it was actually a CF issue. Worth noting for any folks that have a similar problem.
